source/ngf-mtl/impl.mm

```diff
@@ -1781,14 +1842,18 @@ ngf_error ngf_cmd_begin_render_pass_simple_with_sync(
       clears[i].clear_color[1] = clear_color_g;
       clears[i].clear_color[2] = clear_color_b;
       clears[i].clear_color[3] = clear_color_a;
+      store_ops[i]             = rt->attachment_descs.descs[i].sample_count > NGF_SAMPLE_COUNT_1
+                                     ? NGF_STORE_OP_RESOLVE
+                                 : rt->attachment_descs.descs[i].is_sampled ? NGF_STORE_OP_STORE
+                                                                            : NGF_STORE_OP_DONTCARE;
     } else if (rt->attachment_descs.descs[i].type == NGF_ATTACHMENT_DEPTH) {
       clears[i].clear_depth_stencil.clear_depth   = clear_depth;
       clears[i].clear_depth_stencil.clear_stencil = clear_stencil;
+      store_ops[i] =
```

Comment:
`store_op` should be set the same way for both color and render targets:

IF the rt has any resolve attachments, all multisampled attachments should have STORE_OP_RESOLVE, otherwise the resolve op for multisampled attachments is decided the same way as for non-multisampled.

For non-multisampled attachments, if `is_sampled` is true then `store_op` should be STORE, otherwise DONTCARE.

Ensure that the behavior is the same in the vulkan version as well (it's not currently).